### PR TITLE
refactor landpage

### PR DESCRIPTION
**Pull request by JatuFaique**
Added and Modified to Landing Page

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,11 +1,78 @@
 import React from "react";
+import Logo from "../assets/OfficeuiLogo.png";
 
 export default function Footer() {
   return (
-    <div className="section__footer px-3">
-      <div className="section__about-us p-1-25">Office Bay</div>
+    <div className="py-4" id="footer">
+      <div class="foot-col-1">
```
Comment (govindappaarun) [inline src/components/Footer.js:7]: I think you wanted to give className? instead of class

Comment (govindappaarun) [reply src/components/Footer.js:7]: I see many places in this file by miss you have put prop as class

File: src/styles/styles.css [hunk 2]
```diff
@@ -128,18 +128,110 @@
...
+  background: #63636f;
+  color: #dbdbdb;
+}
+
+#footer .logo img {
+  width: 50px;
+  float: left;
+}
+
+#footer ul li a {
```
Comment (govindappaarun) [inline src/styles/styles.css:189]: suggestion: avoid selector with too much specificity

File: src/pages/Home.js [hunk 3]
```diff
@@ -27,6 +27,35 @@ function Home() {
...
+            <br />
+            work from home enivronment.
+          </div>
+        </div>
+        <div className="promo__images p-2">
+          <div className="promo__image">
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTjNhE17d0VzkBRvRx4OIm0jjm_SR6ByRY4Zg&usqp=CAU" />
+          </div>
+          <div className="promo__image grid-span-2">
+            <img src="https://brandongaille.com/wp-content/uploads/2013/12/Bassett-Company-Logo.jpg" />
```
Comment (govindappaarun) [inline src/pages/Home.js:46]: I would suggest you use images from your own cdn, like cloudinary than from the source

**Review by govindappaarun — COMMENTED** [review]
I would suggest to fix those minor things before you finish the PR

File: src/components/Footer.js (hunk 1 above)
Comment (naazneen) [inline src/components/Footer.js:7]: Please correct these.

**Review by naazneen — APPROVED** [review]
Few corrections, otherwise the page looks good.
Good to merge

**Pull request merged**